apiary.apib

```diff
@@ -2425,6 +2406,173 @@ The actual data must be provided in one of the following attributes:
             }
         }
 
+### Create new table from snapshot asynchronously [POST]
```

Comment:
Jak jsem to rozdelil na 4 fiktivni endpointy, tak nastava problem s duplicitou url :-/

> action with method 'POST' already defined for resource '/v2/storage/buckets/{bucket_id}/tables-async'
> Line: 2409• API Blueprint

Nevim, jestli to jde, udelat ruzne definice stejneho endpointu?

Pripadne by pomohlo udelat proste "nove" endpointy ve stylu `.../tables-async-file`, `.../tables-async-snapshot` atp. - coz mi prijde asi nejlepsi reseni. (podobne jsme to udelali u CreateBucket.

Reply:
IMHO je to jen warning, ne error, ne?

Reply:
Domluveno, ze warning nevadi, vizualne to vypada OK a jde to proklikavat jednotlive.